packages/dnb-design-system-portal/src/shared/parts/PropertiesTable.tsx

```diff
@@ -14,9 +13,56 @@ const components = {
 const StyledTable = styled(Table)`
   td {
     white-space: nowrap;
+    // vertical-align: middle !important;
   }
 `
 
+const colorValue = 'var(--color-success-green)'
+const colorString = 'var(--color-fire-red)'
+const colorType = 'var(--color-violet)'
+const colorUndefined = 'var(--color-black-55)'
+
+const DocCode = ({
```

Comment:
```suggestion
const FormattedCode = ({
```

I think, something more descriptive would be nice.